solution_1.rb

```diff
@@ -0,0 +1,91 @@
+class Tutorial
+
+  def solution(x)
+    str = x.to_s
+    num = str.chars.sort.join.reverse!
+  end
+
+  def solution_3(a, b, c)
+    mnum = [a, b, c].max
+    if mnum == a
+      i = [2, a].min
+      str = 'a' * i
+      a -= i
+    elsif mnum == b
+      i = [2, b].min
+      str = 'b' * i
+      b -= i
+    else
+      i = [2, c].min
+      str = 'c' * i
+      c -= i
+    end
```

Comment:
Maybe we can implement this without using same logic for a, b, c case like

```
nums = [a, b, c]
max_num = [a, b, c].max
max_char = %w[a b c][nums.index(max_num)]
i = [2, max_num].min
str = max_char * i
nums[%w[a b c].index(char)] -= i
```

and maybe the below section can do this too.